spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -544,6 +544,41 @@ def update_price_tier!(app_id, price_tier)
       handle_itc_response(r.body)
     end
 
+    def transform_to_raw_pricing_intervals(app_id = nil, purchase_id = nil, pricing_intervals = nil, subscription_price_target = nil)
```

Comment:
Can you explain me why this method belongs to the a client instance instead of a class method in  `iap_detail`?

Reply:
Sure. Even though `iap.rb` requires `iap_detail.rb`, the class `IAP` doesn't depend on `IAPDetail`. And it makes sense to me because I don't see an obvious connection between those classes.
The only place I could think that can be accessed by both classes was `TunesClient`. But then again I'm not that familiar with the codebase, is there a better place to put it?